Fix typo: 'holday' -> 'holiday'

rjc opened the pull request:
While there, be consistent when using hyphens.

Commit:
Fix typo: 'holday' -> 'holiday'

While there, be consistent when using hyphens.

rjc commented:
Hello,

Also, I wasn't sure about _non-English users_ as it can be interpreted literally as _users who are not English_ - personally, I would have used _non-English speakers_.

tbabej commented:
Hi @rjc , thanks for the patch! I agree with the "non-English speakers" suggestion, if you wish to update your branch, let me know! :slightly_smiling_face:

Commit:
Use 'speakers' instead of 'users'.

"Non-English users" sounds very much like "users who are not English".
Also, "Non-English speakers" is already a well established phrase.

rjc commented:
@tbabej Thanks, now done.
